src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -4056,7 +4056,7 @@ citus_internal_database_command(PG_FUNCTION_ARGS)
 	}
 	else
 	{
-		ereport(ERROR, (errmsg("citus_internal_database_command() can only be used "
+		ereport(ERROR, (errmsg("citus_internal.database_command() can only be used "
```

Comment:
The line is covered, see [src/test/regress/expected/create_drop_database_propagation.out](https://github.com/citusdata/citus/pull/7470/files#diff-51744aa2356f2516b6e5ae92e2e1e804a67bed8e55df6b3fb607d0f4a0bab52e) line 27-28

Reply:
codecov has been a bit unreliable recently, reporting things as uncovered even though they are. 

In either case, this isn't really a new line so no need to add coverage for it in this PR.